src/core/assets/BrainIcon.tsx

```diff
@@ -0,0 +1,26 @@
+export const BrainIcon = (props: React.SVGProps<SVGSVGElement>) => (
```

Comment:
would https://lucide.dev/icons/brain work instead? can just 
```import { Brain } from 'lucide-react'``` since we already use that library

Reply:
I think the other history PR removes that library and its trash icon.

Reply:
Yeah I was working on avoiding using a library if we don't have to so we're not tied to a upgrades of packages.